proactive-node-hpa.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  labels:
+    app: proactive
+  name: proactive-node
+spec:
+  maxReplicas: 4
+  minReplicas: 1
+  scaleTargetRef:
+    apiVersion: v1
+    kind: DeploymentConfig
+    name: proactive-node
+  targetCPUUtilizationPercentage: 50
```

Comment:
Just a question. Why is it 50 percent?

Reply:
Why... why not?
It is related to the average value and I didn't really played with that value to figure what's the best number (there is probably none that is the best fit for all kind of jobs).